microservices/pcs-simulation-webui/scripts/docker/build.cmd

```diff
@@ -3,10 +3,14 @@
 @ECHO off
 setlocal
```

Comment:
why does this file not enableExtensions or enableDelayedExpansion in its setLocal call?

Reply:
Unknown. This change was not made by me; this is the state that the code was in when it was in its original repo.